app.yml

```diff
@@ -180,6 +180,19 @@ services:
       networks:
         - backend
 
+  redis:
+    # Eventually we'll probably want to use redis/redis-stack-server:7.4.0-v1 here,
+    # which excludes the RedisInsight GUI front-end.
+    image: redis/redis-stack:7.4.0-v1
+    environment:
+      REDIS_ARGS: '--maxmemory 4GB --maxmemory-policy allkeys-lru'
+    ports:
+      - '6379:6379'
+      - '8001:8001'
```

Comment:
We'll need to clarify the situation around the ports before deploying this in prod. I don't think they're open due to the firewall sitting in front of it but it needs double checking.

We should be able to connect to the 8001 port for the insights GUI via an ssh tunnel/port forward. I did that i the past but not in a while - need to dig up the commands again.

Reply:
If it’s a swarm, there may need to be firewalls opened between the hosts - I don’t think it’s available in prod based on the current network security groups.

Reply:
All the swarm hosts are in the same NSG and I'm pretty sure we only need to open 8001 (to Bastion) in case we want to use the Redis GUI.

I've verified that this is working as intended on dev (which is technically also a swarm albeit with just one node).